src/naarad/metrics/jmeter_metric.py

```diff
@@ -293,7 +287,7 @@ def calculate_stats(self):
 
   def plot_timeseries(self, graphing_library='matplotlib'):
     if graphing_library != 'matplotlib':
-     return Metric.plot_timeseries(self, graphing_library)
+      return Metric.plot_timeseries(self, graphing_library)
```

Comment:
Yep, it seems have 2-char indent for the styling format. It was only one before.